Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM ubuntu:trusty
 MAINTAINER "Syncano DevOps Team" <devops@syncano.com>
 
-RUN apt-get update
-RUN apt-get upgrade -y
+ENV LAST_REFRESHED 2016-03-22
+ENV SYNCANO_APIROOT https://api.syncano.io/
```

Comment:
This env var could also go a little lower. Maybe add some comments like:
`--- CUT ---` ? to mark the parts that are common across all images? ;)